test/e2e/tests/conftest.py

```diff
@@ -125,8 +125,8 @@ def pytest_configure(config):
                 },
                 "task": {
                     "jobs": 10 ** 2,
-                    "concurrency": 4,
-                    "submit_timeout": 200,  # measured in seconds
+                    "concurrency": 4,  # task-retrieving concurrency
```

Comment:
Would it be preferred to set this to 1 here rather than hard coding it in the other file?

Reply:
If we set this to 1, then the task-retrieving process will be sequential and not parallel (that should stay that way). I think we can add another field though. One for the task-submitting part and another for the task-retrieving part.

Then again, not too concerned about this.